### DISCUSSION TITLE
How do you test A2000 specific accelerators/expansions?....

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1509] [Q&A] [answered]
....when we no longer support that Amiga model? (and we don't do A1500)...

I ask because I *think* A2620/A2630 accelerator support is broken (can recreate), however I hesitate hoisting a ticket about it, because it seems to me, these accelerators are pretty pointless. if you cannot build the Amiga machine they were designed for?

TIA

**Comment by midwan** [comment, marked as answer]
The A2000 was basically an expanded A500.
I guess you could start with an A500 config from quickstart, then change the Chipset options to A2000 and add expansions as needed.

**Reply by giantclambake** [reply]
Ahh, ok, thanks for that ~ I had a more hw orientated mindset going.. cards = slots = downgrade an A3000? ...haha.. cheers ;)
